config/webpack.config.js

```diff
@@ -403,6 +404,12 @@ module.exports = async (env = {}) => {
       },
     },
     plugins: [
+      new webpack.DefinePlugin({
```

Comment:
We're already using the `DefinePlugin` in the config, is there a reason this variable wasn't added to the existing plugin?

Reply:
@theodur The reason is that I didn't get enough sleep. Good catch. I've updated.